### PR TITLE
APP-916: Update dependencies

### PR DESCRIPTION
**Pull request by hugokallstrom**
### Checklist

- [ ] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -7,5 +7,5 @@ repositories {
 }
 dependencies {
     implementation("com.android.tools.build:gradle:7.0.0")
```
Comment (hedvigoscar) [inline buildSrc/build.gradle.kts:9]:
```suggestion
    implementation("com.android.tools.build:gradle:7.0.2")
```

File: buildSrc/src/main/kotlin/Libs.kt [hunk 2]
```diff
@@ -89,17 +88,17 @@ object Libs {
         const val tracking = "com.google.firebase:firebase-analytics-ktx"
     }
 
-    const val mixpanel = "com.mixpanel.android:mixpanel-android:5.9.1"
+    const val mixpanel = "com.mixpanel.android:mixpanel-android:6.0.0-beta1"
```
Comment (hedvigoscar) [inline buildSrc/src/main/kotlin/Libs.kt:91]: Is this safe? 😬

**Review by hedvigoscar — APPROVED** [review]
Approved with nits

File: buildSrc/src/main/kotlin/Libs.kt (hunk 2 above)
Comment (hugokallstrom) [reply buildSrc/src/main/kotlin/Libs.kt]: Moved to stable release

**Pull request merged**